### DISCUSSION TITLE
Versioning of content-docs plugin

### DISCUSSION THREAD

**Discussion by Sotty75** [facebook/docusaurus#5993] [Q&A] [answered]
Hi,
in our site we need to maintain two different apis with two different versioning patterns.
One APIs is stored in the standard document library and the versioning was fine, with current version 3.11.
The new API is supposed to be stored in a separate doc library added as a plugin and starting from version 1.

```
 [
            '@docusaurus/plugin-content-docs',
            {
                id: 'new-api-doc', // omitted => default instance
                path: 'new-api-doc',
                routeBasePath: 'new-api-doc',
                sidebarPath: require.resolve('./sidebars-new-api-doc.js'),
                // ... other options
            },
        ],
```

is docusaurus supporting versiong for plugin content docs?
Is there any configration example?

**Comment by Josh-Cena** [comment, marked as answer]
Versioning is done on a per-plugin basis, so your extra doc plugin can be versioned as well. The doc plugin in the preset is exactly the same as the standalone one, so you could configure them in the same way.
